pkg/api/alert/v1/types.go

```diff
@@ -40,21 +40,24 @@ type Alert struct {
 
 // AlertSpec is the spec for a Alert resource
 type AlertSpec struct {
-	Namespace         string `json:"namespace,omitempty"`
-	Registry          string `json:"registry,omitempty"`
-	ImagePath         string `json:"imagePath,omitempty"`
-	AlertImageName    string `json:"alertImageName,omitempty"`
-	AlertImageVersion string `json:"alertImageVersion,omitempty"`
-	CfsslImageName    string `json:"cfsslImageName,omitempty"`
-	CfsslImageVersion string `json:"cfsslImageVersion,omitempty"`
-	BlackduckHost     string `json:"blackduckHost,omitempty"`
-	BlackduckUser     string `json:"blackduckUser,omitempty"`
-	BlackduckPort     *int   `json:"blackduckPort,omitempty"`
-	Port              *int   `json:"port"`
-	StandAlone        *bool  `json:"standAlone"`
-	AlertMemory       string `json:"alertMemory,omitempty"` // Should be passed like: e.g "1300Mi"
-	CfsslMemory       string `json:"cfsslMemory,omitempty"` // Should be passed like: e.g "1300Mi"
-	DesiredState      string `json:"desiredState,omitempty"`
+	Namespace         string   `json:"namespace,omitempty"`
+	Registry          string   `json:"registry,omitempty"`
+	ImagePath         string   `json:"imagePath,omitempty"`
+	AlertImageName    string   `json:"alertImageName,omitempty"`
+	AlertImageVersion string   `json:"alertImageVersion,omitempty"`
+	CfsslImageName    string   `json:"cfsslImageName,omitempty"`
+	CfsslImageVersion string   `json:"cfsslImageVersion,omitempty"`
+	Port              *int     `json:"port"`
+	StandAlone        *bool    `json:"standAlone"`
+	Environs          []string `json:"environs,omitempty"`
+	PVCStorageClass   string   `json:"pvcStorageClass"`
+
+	// Should be passed like: e.g "1300Mi"
+	AlertMemory string `json:"alertMemory,omitempty"`
+	CfsslMemory string `json:"cfsslMemory,omitempty"`
+	PVCMemory   string `json:"pvcMemory"`
```

Comment:
This isn't really Memory, it's storage space or PVCSize or something.  Memory is kind of misleading.  Suggest renaming PVCMemory to something that indicates this is a disk space/size requests in all places similar is used.

Reply:
@mphammer For PVC, pls refer to Black Duck and use the same naming convention.

Reply:
There is now PVCStorageClass, PVCSize, and PVCName like the PVC struct in Blackduck.